backend/chainlit/data/sql_alchemy.py

```diff
@@ -7,7 +7,6 @@
 
 import aiofiles
 import aiohttp
-
```

Comment:
Why remove this whitespace? I feel it's often indicative to separate:
1. Python stdlib imports
2. Third-party deps
3. Internal deps

Reply:
That is the result of running `ruff format`.